lib/matplotlib/backend_bases.py

```diff
@@ -221,7 +221,7 @@ def draw_path_collection(self, gc, master_transform, paths, all_transforms,
         behavior of :meth:`draw_path_collection` can be made globally.
         """
         path_ids = [
-            (path, transforms.Affine2D(transform))
+            (path, transform)
```

Comment:
AFAICS the list comprehension does nothing after the change, so it could be removed.

Reply:
I am not sure about it, I am open for suggested change.

Reply:
Pretty sure you can replace `path_ids` in the call to `self._iter_collection()` directly with:
```python
self._iter_collection_raw_paths(master_transform, paths, all_transforms)
```
and eliminate the list comprehension. This may or may not buy a little more performance.

Reply:
or at least:
~~~
path_ids = self._iter_collection_raw_paths(
    master_transform, paths, all_transforms)
~~~

if we don't want to bloat the call of `self._iter_collection()`.